src/lib/list.ts

```diff
@@ -4,7 +4,12 @@ export async function list(this: Connection): Promise<TableMetadata[]> {
   const address = await this.signer.getAddress();
 
   const resp: TableMetadata[] = await fetch(
-    `${this.host}/tables/controller/${address}`
+    `${this.host}/tables/controller/${address}`,
```

Comment:
In the v2 branch of the Valdator this bearer token is required.  See #43 for more detail